server/Controllers/PointsController.cs

```diff
@@ -117,26 +110,30 @@ public IActionResult PostOne(Guid id, [FromBody]Point point)
                 _logger.LogWarning($"Overwriting the point with id {point.Id}");
             }
             points[id] = point;
-            WriteStorage(points);
-            return Json(points.Values);
+            WriteStorage(points, tags);
+            return Json(points.Values.Select(p => p.ToJson(tags)).ToList());
```

Comment:
returns all the values, however it seems it should be the only one :) has been talked offline